Add format description in run doc

hqhq opened the pull request:
And only set false to --oom-kill-disable, we just need the default
value to show.

Signed-off-by: Qiang Huang h.huangqiang@huawei.com

moxiegirl commented on docs/reference/run.md:
```diff
@@ -505,18 +505,18 @@ parent group.
-| `--oom-kill-disable=true` or `false` | Whether to disable OOM Killer for the container or not.                                     |
-| `--memory-swappiness=""  `           | Tune a container's memory swappiness behavior. Accepts an integer between 0 and 100.        |
+| Option                     |  Description                                                                                   |
+|----------------------------|------------------------------------------------------------------------------------------------|
```
@hqhq thank you for the contribution.  You need to use `tics` instead of angle brackets to show the format.  The angle brackets are not rendering properly.

![image](https://cloud.githubusercontent.com/assets/1347209/9282457/5c20fbaa-4281-11e5-8a9d-d31e8414e62c.png)

hqhq committed:
Add format description in run doc

And only set false to --oom-kill-disable, we just need the default
value to show.

Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

hqhq commented:
@moxiegirl Updated, also changed format as https://github.com/docker/docker/pull/14006#discussion_r37147754 suggested.

moxiegirl commented:
LGTM Thank you @hqhq ... @thaJeztah for review.

thaJeztah commented:
> also changed format as #14006 (comment) suggested.

:+1: thanks!


LGTM, thank you @hqhq !
